app/src/main/res/menu/cab_conversations.xml

```diff
@@ -42,6 +42,11 @@
         android:icon="@drawable/ic_edit_vector"
         android:title="@string/rename_conversation"
         app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/cab_conversation_details"
+        android:icon="@drawable/ic_info_vector"
+        android:title="@string/conversation_details"
+        app:showAsAction="ifRoom" />
```

Comment:
Let's make it `never`.

```suggestion
        app:showAsAction="never" />
```

Reply:
I see it's `ifRoom` in the original `menu_thread.xml` too, would if good if we set it to `never` there as well.